src/assets/scss/maps/_z-index-map.scss

```diff
@@ -62,4 +62,6 @@ $z-index-map: (
   'modal-error-backdrop-on-search-popup': 1061,
   'modal-error-on-search-popup': 1066,
 
+  // higher than dropdown
+  'multi-form-input-row': 2
```

Comment:
Please move it up there. The values in this map are sorted for readability purposes.